src/Config/Config.php

```diff
@@ -22,8 +22,18 @@ public function getEndpoint(): string
         return $this->endpoint;
     }
 
+    public function getEndpointJson(): array
```

Comment:
strange naming convention "json" to return an array ?

Reply:
maybe this should be getEndpoint and the other funcion getEndpointRaw ?
Or call this function getEndpointDecoded

Reply:
Renamed it too getEndpointFromJson maybe in the feature we create endpoint from yaml, xml, php :)